.travis.yml

```diff
@@ -64,40 +35,13 @@ before_script:
   - travis_retry npm install -g gulp
   - travis_retry npm install karma-phantomjs-launcher karma-phantomjs-shim
   - npm run build
-  - python __init__.py &
-
-  # Install API and run in background
-  - travis_retry git clone https://github.com/18F/openFEC
-  - cd openFEC
-  - git checkout $FEC_BRANCH
-  - psql -c 'create database cfdm_test' -U postgres
-  - pg_restore --dbname cfdm_test data/subset.dump --no-acl --no-owner
-  - travis_retry pip install -r requirements.txt
-  - travis_retry pip install -r requirements-dev.txt
-  - python manage.py update_all --processes 1
-  - python manage.py runserver &
-  - cd ..
 
 script:
-  # Run Python unit tests
-  - if [[ -z $FEC_SAUCE_BROWSER ]]; then py.test; fi
-  # Run Javascript unit tests
-  - if [[ -z $FEC_SAUCE_BROWSER ]]; then npm run test-single; fi
-  # Run Selenium tests if not pull request
-  - if [[ $TRAVIS_PULL_REQUEST = 'false' && ! -z $FEC_SAUCE_BROWSER  ]]; then py.test --selenium; fi
+  - py.test
+  - npm run test-single
 
 after_success:
   # Deploy to appropriate Cloud Foundry space after all builds succeed
   # See `tasks.deploy` for details
-  - python travis_after_all.py
-  - export $(cat .to_export_back)
-  - |
-      if [[ $BUILD_LEADER = 'YES' && $TRAVIS_PULL_REQUEST = 'false' ]]; then
-        if [[ $BUILD_AGGREGATE_STATUS = 'others_succeeded' ]]; then
-          echo "All builds passed; deploying..."
-          invoke deploy --branch $TRAVIS_BRANCH --yes
-        else
-          echo "One or more builds failed; skipping deploy"
-        fi
-      fi
-  - if [[ $BUILD_LEADER = 'YES' ]]; then codecov; fi
```

Comment:
Oh wow, so lots of extra setup and legwork just to support these that we no longer have to worry about with this change. :+1: